README.md

```diff
@@ -1,2 +1,88 @@
-# yotas
-Oss's monetisation system
+# Yotas
+
+Oss's reward system
+
+[French](#french)
+
+[English](#english)
+
+# english
+
+Yotas is OSS's reward system, we would like our open source contributors across the organisation to be rewarded for the effort and energy they put into the platform. Each interaction they have on GitHub will give them a certain number of yotas which they can exchange for goodies in an OSS shop, mentoring, ebooks, etc....
+
+## Why?
+Contributing to open source takes some effort, and we would like every contributor to be rewarded for the time and energy they put into helping our community.
+
+## Description
+This reward system is strongly inspired by the correction points and wallet points system applied at Ecole 42. Correction points at Ecole 42 are points that students earn by correcting other students' projects, and lose when they have other students correct their projects. Wallet points are earned when a user contributes to the community, and wallets are directly redeemable for gifts such as 42 logo t-shirts, jacuzzi and other fun rewards.
+
+The yota will be a reward related to the organisation, which the user will get by creating problems, reviewing code, merging their pull requests, etc. They will be able to use these well earned yotas to buy things in our shop like a call with the project maintainer, t-shirts, internet connection (requires real money), time in a coworking space (requires partnership) and many more.
+
+## How it works
+
+Yotas is a system that allows open source organisations to give rewards to their contributors according to the resources they have.
+
+Here is the schema:
+
+- I am an organisation X with Y resources 
+- I want to redistribute Y to active contributors. 
+- Each time a person contributes, he earns yotas, and with our Y resources we set up a shop where the contributor buys in yotas.
+
+**The purpose of yotas is not to convert the contribution into money.**
+
+It is not a way to fund an open source project as well, but it can allow to use the funding received to reward the contributors.
+
+Github sponsors and Yotas can be used together as follows:
+
+- Organisation X receives funding and therefore has Y resources
+- With these resource it creates a shop where it puts items that its contributors can buy in Yotas
+
+What can be put in a shop on yotas can be anything and everything (mentoring, tutorials, goodies,...) it is up to the organisation to decide according to what it has. 
+
+For OSS, the first step is to offer mentoring to juniors, and we try to have affordable issues so that they can contribute. But we are still thinking about what else we can offer, especially to seniors.
+
+## Note
+This is a web project, which will be heavily based on a GitHub application that will monitor all events in the organisation and award interaction points.
+
+
+# french
```

Comment:
Please if you can reflect suggested corrections from the English to the French translations, it will be awesome!! 

:+1: For taking time to write all this out, you are awesome!

Reply:
done

it's only based on our yesterday's discussion
i havent invent anything